internal/core/arg_specs.go

```diff
@@ -84,6 +84,10 @@ type ArgSpec struct {
 	// EnumValues contains all possible values of an enum.
 	EnumValues []string
 
+	// BadEnumWarning is the warning printed when the value is not in the EnumValues
+	// If not set, the value has to be in EnumeValues
+	BadEnumWarning string
```

Comment:
I would prefer an `AllowUnknownEnumValue bool`

Reply:
Works for me, what about the warning message?